script/encrypt-heroku-auth-token

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -e
+
+function cleanup_at_exit {
+  # remove temporary key file
+  rm tmp.pem
+}
+
+trap cleanup_at_exit EXIT
+
+# Get the repo name from the `origin` git remote.
+export REPO=$(git remote get-url origin | \
+    sed 's/^.*[\/:]\([^\/:]*\)\/\([^\/]*\)$/\1\/\2/' | \
+    sed 's/^\(.*\)\.git$/\1/'
+)
+
+# Get the Travis public key for this repo.
+export TRAVIS_KEY=$(curl --silent https://api.travis-ci.org/repos/$REPO/key)
+
+# Save the key to a file.
+node -e "console.log($TRAVIS_KEY['key'])" > tmp.pem
+
+# Generate a Heroku token, encrypt and encode it and print it in YAML.
+echo -n $(heroku auth:token) | \
+    openssl rsautl -encrypt -pubin -inkey tmp.pem | \
```

Comment:
It would be cool if we didn't need to write a file to invoke `openssl rsautl`, but of course that depends on whether the CLI supports using a string argument.

Reply:
I did expend some effort to avoid this, but was stymied. I also thought it would be good to show the resource cleanup via `trap`.  I would not object to an update avoiding it if it is possible.

Reply:
It's not very high priority, certainly.